content/docs/2.2/scalers/openstack-metric.md

```diff
@@ -0,0 +1,170 @@
++++
+title = "Openstack Metric"
+layout = "scaler"
+availability = "v2.3+"
+maintainer = "Community"
+description = "Scale applications based on the metrics collected from an specific resource."
```

Comment:
Can you provide a more clear description, please?

Reply:
sure!

Reply:
Hi, I updated this description and linked the scaler PR to this doc PR.